### DISCUSSION TITLE
Is Inpainting also affected by the type of model?

### DISCUSSION THREAD

**Discussion by lgs777** [lllyasviel/Fooocus#2136] [Q&A] [answered]
Does the type of model affect outpainting? I am changing the model when doing outpainting. I am curious if the feel of outpainting changes with the model. Or does a different model operate when doing outpainting?

**Comment by mald55** [comment]
how does one use a different inpaint model? can a 1.5 SD model be used?

**Comment by mashb1t** [comment, marked as answer]
Yes, the style will differ. There is no specific in- or outpainting model select, just one model + refiner select. Feel free to use the base model default juggernaut with a 1.5 refiner model plus set refiner switch to 0.1

**Reply by lgs777** [reply]
Thanks for the detailed answer!

**Reply by mald55** [reply]
Thanks for the quick reply.

Quick follow up, is there any point in using a refiner when inpainting if the SDXL custom model being used is good enough?

**Reply by mashb1t** [reply]
I mean, you answered the question already yourself: if the quality is good then no, there isn't.
